docs/pages/installation/windows.md

```diff
@@ -145,7 +145,7 @@ If using Scoop, run the following in the PowerShell to install a Kubernetes clus
   <pre class="codeblock-pre">
   <div class="codeblock"><div class="clipboardjs">scoop install k3d
 k3d cluster create
-export KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"</div></div>
+set KUBECONFIG="$(k3d kubeconfig get 'k3s-default')"</div></div>
```

Comment:
```suggestion
 $env:KUBECONFIG = "$(k3d.exe kubeconfig get 'k3s-default')"</div></div>
```

Reply:
@thisiskaransgit, this will work when k3d.exe's path is available in system path. Do we want to give the absolute path of k3d.exe to avoid any issues?

Reply:
@hexxdump, we expect the user to have k3d.exe in their env variables.

Reply:
@hexxdump and @thisiskaransgit, any final comments here? Please this comment resolve if it's good to go.

Reply:
@Chadha93  the contributor has not incorporated these changes in the PR, Will be closing it as I've created a separate [issue](#7792 ) for this